src/Faker/Provider/fr_FR/Person.php

```diff
@@ -45,16 +45,50 @@ class Person extends \Faker\Provider\Person
         'Corinne', 'Danielle', 'Denise', 'Diane', 'Dorothée', 'Édith', 'Éléonore', 'Élisabeth', 'Élise', 'Élodie', 'Émilie',
         'Emmanuelle', 'Françoise', 'Frédérique', 'Gabrielle', 'Geneviève', 'Hélène', 'Henriette', 'Hortense', 'Inès', 'Isabelle',
         'Jacqueline', 'Jeanne', 'Jeannine', 'Joséphine', 'Josette', 'Julie', 'Juliette', 'Laetitia', 'Laure', 'Laurence',
-        'Lorraine', 'Louise', 'Luce', 'Lucie', 'Lucy', 'Madeleine', 'Manon', 'Marcelle', 'Margaux', 'Margaud', 'Margot',
```

Comment:
Can we revert the removals of names?

@fzaninotto suggested to avoid modifying data sets as this might potentially break tests of consumers of this library. Duplication is - if not accidental - so values are more likely yielded from formatters.

For reference, see #1775.

Reply:
I think that duplication, here, is accidental. We can accept the change.